### ISSUE TITLE
Conversation is crashing

### ISSUE THREAD

**Issue by tdruiva** [twstrike/coyim#170]
When opens a conversation and the fingerprint is not verified yet, when is going to show the notification, it crashes:

```
panic: gui: failed load VerifyIdentityNotification: Error on line 28 char 13: 
Element 'child' was closed, but the currently open element is 'interface' 
```

**Comment by juniorz** [comment]
I could not find this error on master. I suspect this is because we published a broken version to the release branch.

I'm gonna generate binaries from master and also from a stable release branch.

**Comment by olabini** [comment]
OK, I'll close this for now, assuming it has been fixed.

**Comment by juniorz** [comment]
It was fixed by 8f699e0 and ba96011 makes sure it wont happen again.
